browser/resources/brave_new_tab_page_refresh/components/app.style.ts

```diff
@@ -6,6 +6,8 @@
 import { color, font } from '@brave/leo/tokens/css/variables'
 import { scoped } from '../lib/scoped_css'
 
+const narrowBreakpoint = '900px'
```

Comment:
Might be over engineering here, but I wonder if we should have a `breakpoints.ts` file, so we use the same breakpoints in different files.

Not a problem now, but just thinking about when there are lots of people working on the ntp.